go/worker/common/committee/node.go

```diff
@@ -651,6 +708,79 @@ func (n *Node) worker() {
 	}
 }
 
+func (n *Node) updatePeriodicMetrics() {
+	boolToMetricVal := func(b bool) float64 {
+		if b {
+			return 1.0
+		}
+		return 0.0
+	}
+
+	labels := n.getMetricLabels()
+
+	n.CrossNode.Lock()
+	defer n.CrossNode.Unlock()
+
+	n.logger.Debug("updating periodic worker node metrics")
+
+	epoch := n.Group.GetEpochSnapshot()
+	cmte := epoch.GetExecutorCommittee()
+	if cmte == nil {
+		return
+	}
+
+	executorCommitteePeers.With(labels).Set(float64(len(cmte.Peers)))
```

Comment:
Actually this is not the right peers to look at. You probably want to see the list of P2P peers? That one should be in `n.P2P.Peers(runtimeID)`.

Reply:
updated